openid_connect_configuration.conf

```diff
@@ -28,6 +28,18 @@ map $host $oidc_jwt_keyfile {
     default "http://127.0.0.1:8080/auth/realms/master/protocol/openid-connect/certs";
 }
 
+map $host $oidc_logout_endpoint {
+    default "http://127.0.0.1:8080/auth/realms/master/protocol/openid-connect/logout";
+}
+
+map $host $oidc_logout_query_params {
+    # Each IdP may use different query params of the $oidc_logout_endpoint. For
+    # example, Amazon Cognito requires `client_id` and `logout_uri`, and Auth0
+    # requires `client_id` and `returnTo` instead of the default query params.
+    default "post_logout_redirect_uri=$redirect_base/_logout&id_token_hint=$session_jwt";
```

Comment:
I think this approach can still confuse the inexperienced user:
1. We cannot replace the `post_logout_redirect_uri` parameter with something of our own, because the `_logout` location will perform additional functions.
2. We already have 2 similar parameters: `post_logout_redirect_uri` and `$oidc_logout_landing_page`.

Perhaps it makes sense to hide `post_logout_redirect_uri=$redirect_base/_logout&`.